src/d3a/constants.py

```diff
@@ -35,3 +35,10 @@
 # Controls how often will event tick be dispatched to external connections. Defaults to
 # 20% of the slot length
 DISPATCH_EVENT_TICK_FREQUENCY_PERCENT = 20
+
+COLLABORATION_ID = ""
```

Comment:
Can we move these constants to `d3a-interface` too?

Reply:
These constants only apply to d3a, therefore I think it is wrong to move them to d3a-interface